src/pages/index.tsx

```diff
@@ -40,6 +40,12 @@ export default function Home() {
   const limitInMs = limit * 60 * 60 * 1000;
 
   if (status === "authenticated") {
+    fetch(`${process.env.VITE_BACKEND}/me`, { headers: { "Authorization": `Bearer ${session?.id_token ?? "no-token-yet"}` } }).then((res) => {
```

Comment:
Hm. This works fine for now, but I don't like the fact that we're doing this explicitly inline.